Taskfile.yml

```diff
@@ -3,13 +3,15 @@
 version: '3'
 
 env:
+  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2023"
   SRC_DIR: "{{.TASKFILE_DIR}}/src"
 
 tasks:
   lint:
     desc: Formatting and linting
     dir: "{{.SRC_DIR}}"
     cmds:
+      - task: has-latest-opslevel-go
```

Comment:
Along with the other comment, I think it's unusual that `task lint` and `task lintfix` include a step to grab the latest opslevel-go. I don't think that this would fall under formatting or linting.